lizafloyd/portfolio_site
# Logo and Train Update

**Description**
> Provide a description of what your changes do.

Adds updated logos for The Lily and The Washington Post (they are now dark) and visited my friend content aware in his photo shop to remove the train door buttons/warning logo. I've saved the train door photo as `_edited.jpg` so you can revert it if you don't prefer it.

**Testing Instructions**
> Give step by step instructions on how to test your changes.

Load up the homepage and observe the changes.

**Additional Notes**
> Anything else that will help us test the pull request.

🦉
